### PR TITLE
chore: add LICENSE file to all npm packages

### PR DESCRIPTION
**Pull request by opichals**
Together with a new npm release of the modified fodlers this fixes https://github.com/benmosher/eslint-plugin-import/issues/1225

### CODE REVIEW

File: memo-parser/LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```
Comment (ljharb) [inline memo-parser/LICENSE]: let's make these gitignored and copied in via prepublish from the root, rather than duplicating the content.

Comment (opichals) [reply memo-parser/LICENSE]: or better yet, what about making this a proper monorepo a use lerna@3 to deal with this.

Comment (ljharb) [reply memo-parser/LICENSE]: absolutely not; for one, we'd have to move all of our packages into some kind of "packages" dir, which is too much git churn; but also in my experience lerna simply does not work well beyond `bootstrap` and `exec`, and i have no interest in using it on additional projects.

**Comment by opichals** [comment]
> let's make these gitignored and copied in via prepublish from the root, rather than duplicating the content.

@ljharb addressed

**Comment by opichals** [comment]
@ljharb Thank you!

**Comment by ljharb** [comment]
Every package should have been released at this point with the license, ftr.

**Pull request merged**